pkgs/applications/video/dmlive/default.nix

```diff
@@ -13,16 +13,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "dmlive";
-  version = "5.3.1";
+  version = "5.3.2";
 
   src = fetchFromGitHub {
     owner = "THMonster";
     repo = pname;
-    rev = "0a07fd1b831bc9e9d34e474284430297b63446c7"; # no tag
-    hash = "sha256-Jvxbdm9Swh8m03uZEMTkUhIHNfhE+N2a3w7j+liweKE=";
+    rev = "3736d83ac0920de78ac82fe331bc6b16dc72b5cd"; # no tag
+    hash = "sha256-3agUeAv6Nespn6GNw4wmy8HNPQ0VIgZAMnKiV/myKbA=";
   };
 
-  cargoHash = "sha256-/84T7K6WUt2Bfx9qdZjyOHcJEGoquCfRX1ctQBuUjEc=";
+  cargoHash = "sha256-66rkD4K55DLArn0a1QkxtbRCqkTxTTHPffIEeXOhQJE=";
```

Comment:
Builds fail because this is wrong: https://github.com/NixOS/nixpkgs/pull/288319/checks?check_run_id=21485757847.